tools/topology/topology2/get_abi.sh

```diff
@@ -11,6 +11,14 @@ cat <<EOF_HEADER
 Object.Base.manifest."sof_manifest" {
 	Object.Base.data."SOF ABI" {
 EOF_HEADER
+
+# ABI major/minor/patch numbers are 2 bytes each
```

Comment:
Please mention IPC3/IPC4 here. The relationship between IPC4 and topology2 is not obvious IMHO.

Reply:
I've added 2 separate abi scripts for IPC3 and IPC4. I dont know if we'll ever use tplg2 for any of our IPC3 topologies but if we do, this will be useful.